src/Const/Commands.ts

```diff
@@ -46,88 +46,102 @@ export interface CommandDefinition {
     usage?: string,
     description?: string,
     permissionLevel: AccessLevel,
+    isCheat: boolean
 }
 
 export interface CommandCallback {
-    (client: Client, ...args: string[]): void 
+    (client: Client, ...args: string[]): string | void 
```

Comment:
In the future we can send this back to the client and display the result in the console

Reply:
yes it will be good. we can say that this is a temporary method. the lack of any output was already killing me lol